backend/routes/db.js

```diff
@@ -0,0 +1,95 @@
+var express = require('express');
+var router = express.Router();
+var fetch = require('node-fetch');
+var json = require('json');
+
+var classData = {};
+var items = {};
+var items2 = {};
+
+const getData=()=>{
+    console.log("get data start");
+    console.time('parse');
+    fetch('https://andromeda.miragespace.net/slugsurvival/data/fetch/terms/2210')
+    .then(res => res.json()) 
+    .then(json => {
+        items = json;
+        getData2();
+      });
+}
+
+const getData2=()=>{
+  fetch('https://andromeda.miragespace.net/slugsurvival/data/fetch/courses/2210')
+  .then(res => res.json())
+  .then(json => {
+    items2 = json;
+    console.log('get data completed');
+    parsedata();
+  })
+}
+
+getData();
+
+function parsedata() {
+  console.log("Started parse data");
```

Comment:
add some inline comments so whoever might need to modify this doesn't have to figure it out themselves